server/src/main/java/se/fortnox/reactivewizard/server/RwServer.java

```diff
@@ -33,38 +32,32 @@
 @Singleton
 public class RwServer extends Thread {
 
-    private static final Logger      LOG                         = LoggerFactory.getLogger(RwServer.class);
-    private static final int         COMPRESSION_THRESHOLD_BYTES = 1000;
-    private static final Set<String> COMPRESSIBLE_MIME_TYPES      = new HashSet<>(asList(
+    private static final Logger LOG = LoggerFactory.getLogger(RwServer.class);
+    private static final int COMPRESSION_THRESHOLD_BYTES = 1000;
+    private static final Set<String> COMPRESSIBLE_MIME_TYPES = new HashSet<>(asList(
         "text/plain",
         "application/xml",
         "text/css",
         "application/x-javascript",
         "application/json"
     ));
 
-    private final  ServerConfig      config;
-    private final  ConnectionCounter connectionCounter;
-    private final  LoopResources     eventLoopGroup;
-    private final  DisposableServer  server;
-    private static Runnable          blockShutdownUntil;
+    private final ServerConfig config;
+    private final ConnectionCounter connectionCounter;
+    private final DisposableServer server;
+    private static Runnable blockShutdownUntil;
 
     @Inject
-    public RwServer(ServerConfig config, CompositeRequestHandler compositeRequestHandler, ConnectionCounter connectionCounter) {
-        this(config, compositeRequestHandler, connectionCounter, null);
-    }
-
-    RwServer(ServerConfig config, CompositeRequestHandler compositeRequestHandler, ConnectionCounter connectionCounter, LoopResources loopResources) {
-        this(config, connectionCounter, createHttpServer(config, loopResources), compositeRequestHandler, loopResources);
+    RwServer(ServerConfig config, CompositeRequestHandler compositeRequestHandler, ConnectionCounter connectionCounter) {
```

Comment:
Removing public here intentionally? Isn't this a breaking change? No problems for Guice to create this?

Reply:
Guice seems fine with this, but it wasn't intentional, so I'll fix.